README.md

```diff
@@ -128,6 +135,8 @@ is useful for project-local configurations.
 
 You can execute tasks from Telescope by running `:Telescope yabs tasks` / `:Telescope yabs current_language_tasks` or `:Telescope yabs global_tasks`.
 
+To enable Telescope integration you need to add `require('telescope').load_extension('yabs')` to your configuraiton.
```

Comment:
Is this actually necessary? I could be wrong but I don't remember adding this to my config...is this something new that telescope added recently?

Reply:
Yes, it always has been this way. You need to load the extension in order to use. You can check out other official Telescope extensions:

* https://github.com/nvim-telescope/telescope-cheat.nvim#installation
* https://github.com/nvim-telescope/telescope-github.nvim#setup

Reply:
I wish soo badly that I had access to my computer right now...I searched in my dotfiles repo and couldn't find any reference to "load_extension". It's possible that either the repo isn't up to date or github search is acting up (which it seems to do quite frequently TBH), but I really don't remember ever adding that line to my config...

Could you try running `:Telescope yabs tasks` without having first run `load_extension` and tell me what happens?

Thanks.

Reply:
> Could you try running :Telescope yabs tasks without having first run load_extension and tell me what happens?

Oh, right. You can run `:Telescope yabs tasks` without `load_extension`. Telescope is smart and it will try to load the extension. But in order to have a completion I usually put `load_extension` in my config.

Reply:
Got it. That makes sense.

It did seem strange that I wasn't getting tab completion myself actually...now I know!

Reply:
Maybe change this to something like:

"If you use the telescope integration, it is recommended that you add this line to your config to load the extension at startup (this enables tab completion, etc.)"

or something similar.

Reply:
> It did seem strange that I wasn't getting tab completion myself actually...now I know!

You will have the completion, but only after first `:Telescope yabs something` call

Reply:
> Maybe change this to something like:

Changed!